src/univariate/discrete/binomial.jl

```diff
@@ -73,6 +73,22 @@ function mode(d::Binomial{T}) where T<:Real
 end
 modes(d::Binomial) = Int[mode(d)]
 
+function median(rv::Binomial)
```

Comment:
Maybe use
```suggestion
function median(d::Binomial)
```
or

```suggestion
function median(dist::Binomial)
```

to make it more consistent with the other functions?